src/components/Vuetify/util/goTo.js

```diff
@@ -74,7 +74,12 @@ export default function goTo (target, options) {
     let targetPosition = Math.floor(startLocation + distanceToScroll * easingFunction(progressPercentage))
 
     window.scrollTo(0, targetPosition)
-    if (Math.round(window.pageYOffset) === targetLocation) return
+
+    if (
+      Math.round(window.pageYOffset) === targetLocation ||
+      progressPercentage === 1
```

Comment:
First condition could be removed (it will behave wrong for easing function like http://easings.net/en#easeInOutElastic - i'm aware that it's not in the scope of this issue) and `===` could be changed to `>=`

Reply:
I wanted to introduce as little changes to the original code as possible, but what you're saying makes sense. If it's no issue with you guys I can make that modification in this same PR.

Regarding the `>=` though that should never trigger because of the `Math.min` a bit above that part. The maximum value of `progressPercentage` is effectively 1.

Reply:
You're right, i missed the `min` part